apps/shell/views/index.hbs

```diff
@@ -3,21 +3,73 @@
 
 <head>
   <title>Shell</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="{{baseURI}}/javascripts/jquery-3.0.0.min.js"></script>
+      <script src="{{baseURI}}/javascripts/bootstrap.min.js"></script>
+     <link rel="stylesheet" href="{{baseURI}}/stylesheets/bootstrap-theme.min.css">
+    <link rel="stylesheet" href="{{baseURI}}/stylesheets/bootstrap.min.css">
   <link rel="stylesheet" href="{{baseURI}}/stylesheets/style.css">
 
   <script src="{{baseURI}}/javascripts/hterm_all_1.85.js"></script>
   <script src="{{baseURI}}/javascripts/ood_shell.1.js"></script>
+  <script src="{{baseURI}}/javascripts/themes.js"></script>
 </head>
 
 <body>
-  <div id="terminal"></div>
+  <div id="terminal">
+    
+    <nav class="navbar navbar-inverse navbar-static-top navbar-expand" role="navigation">
+      <div class="container-fluid" >
+      <ul class="navbar-nav left-padding-none">
+            <li class="nav-item custom-host">
+            <label id="host" class="label">Host: {{host}} </label>
+          </li>
+      </ul>
+            <ul class="navbar-nav left-padding-some">
+          <li class="nav-item hidden-xs">
+            <label id="directory" class="label hidden-xs"></label>
+          </li>
+      </ul>
+          <ul class="navbar-nav form-inline navbar-right left-padding-none">
+          <li class="nav-item">
+            <div class="form-group custom-form">
+            <label class="label" for="colors">Themes:</label>
+          <select id="colors" class="form-control custom-select">
+        <option value="default">Default</option>
+        <optgroup label="Dark">
+        <option value="Monokai">Monokai</option>
+        <option value="Pastel Dark">Pastel Dark</option>
+        <option value="Cobalt">Cobalt</option>
+        <option value="Tomorrow Night Blue">Tomorrow Night Blue</option>
+        <option value="Tomorrow Night Bright">Tomorrow Night Bright</option>
+        <option value="Tomorrow Night Eighties">Tomorrow Night Eighties</option>
+        <option value="Tomorrow Night">Tomorrow Night</option>
+        <option value="VibrantInk">Vibrant Ink</option>
+        <option value="idle Fingers">Idle Fingers</option>
+        <option value="Solarized Dark">Solarized Dark</option>
+      </optgroup>
+        <optgroup label="Bright">
+        <option value="Solarized Light">Solarized Light</option>
+        <option value="CLRS">CLRS</option>
+        <option value="Tomorrow">Tomorrow</option>
+        <option value="Light Background">Light Background</option>
```

Comment:
Lets reorder these alphabetically. So Default can still be the first item, then Bright optgroup, finally Dark optgroup. In each optgroup, order alphabetically (ignoring case).

Also just realized maybe we should get rid of `<option value="Light Background">Light Background</option>` unless that theme is actually MIT like the others. The iTerm2 project is not MIT but GPL. The other bright colors work great.